graphql/fragments.ts

```diff
@@ -44,6 +45,10 @@ export const STANDARD_PRODUCT_BASIC_FIELDS_FRAGMENT = graphql(`
       id
       name
     }
+    sizeRange {
+      id
+      label
```

Comment:
FYI the SizeRange type also has a `name` field in the API serving as alias for `label`